Commit 726d7e6a4fd293ec26509d75f10102ebe820d7cc breaks compilation

After commit 726d7e6a4fd293ec26509d75f10102ebe820d7cc I get this error
```
../es-core/src/renderers/Renderer_GLES10.cpp: In function 'void Renderer::setMatrix(const Transform4x4f&)':
../es-core/src/renderers/Renderer_GLES10.cpp:196:17: error: variable 'Transform4x4f matrix' has initializer but incomplete type
   Transform4x4f matrix = _matrix;
                 ^~~~~~
```
Once I revert that commit it compiles without a problem

--- a/es-core/src/renderers/Renderer_GLES10.cpp
+++ b/es-core/src/renderers/Renderer_GLES10.cpp
@@ -1,6 +1,7 @@
 #if defined(USE_OPENGLES_10)
 
 #include "renderers/Renderer.h"
+#include "math/Transform4x4f.h"
 #include "Log.h"
 #include "Settings.h"
 

